core/metacling/src/TCling.cxx

```diff
@@ -5198,7 +5208,11 @@ namespace {
 
 Int_t TCling::LoadLibraryMap(const char* rootmapfile)
 {
+   if (rootmapfile && *rootmapfile && !requiresRootMap(rootmapfile, fInterpreter))
```

Comment:
maybe
```
rootmapfile == nullptr || *rootmapfile = '\0' || !requiresRootMap(rootmapfile, fInterpreter))
```

Reply:
`requiresRootMap` assumes we have checked for `rootmapfile && *rootmapfile`

Reply:
I know :) and the proposed new version does that and return early if the rootmapfile string is empty or nullptr.

Reply:
Now I remember, when `rootmapfile == nullptr` we do a scan over all rootmap files. We cannot use the generic condition to exit early which I prefer, too :)